packages/round-manager/src/features/round/CreateRoundPage.tsx

```diff
@@ -50,8 +48,8 @@ export default function CreateRound() {
           <main>
             <FormWizard
               steps={[
-                RoundDetailForm,
-                ApplicationEligibilityForm,
+                // RoundDetailForm,
+                // ApplicationEligibilityForm,
```

Comment:
We probably want this back
```suggestion
                ApplicationEligibilityForm,
```